lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -56,7 +56,8 @@ package rdpclient
 #cgo linux,amd64 LDFLAGS: -L${SRCDIR}/target/x86_64-unknown-linux-gnu/release
 #cgo linux,arm LDFLAGS: -L${SRCDIR}/target/arm-unknown-linux-gnueabihf/release
 #cgo linux,arm64 LDFLAGS: -L${SRCDIR}/target/aarch64-unknown-linux-gnu/release
-#cgo linux LDFLAGS: -l:librdp_client.a -lpthread -ldl -lm -latomic
+#cgo linux LDFLAGS: -l:librdp_client.a -lpthread -ldl -lm
+#cgo linux,arm LDFLAGS: -latomic
```

Comment:
Add this flag to line 57, or if you want to keep it a separate line at least move it up so the linux,arm flags are all together.

Reply:
I was also thinking about that, but unfortunately that fails the build. Linker require to pass libraries in the order they are needed; in this case `librdp_client.a` require `libatomic`, so `libatomic` must go after the `ibrdp_client.a`.
I added a comment to explain that.